### ISSUE TITLE
cardbuilder throws exception when rendering Rage Extractor due to missing icon for {P} symbol

### ISSUE THREAD

**Issue by melvinzhang** [magarena/magarena#598]
```
Exception from controller.runGame: unknown mana cost "{P}"
java.lang.RuntimeException: unknown mana cost "{P}"
        at magic.model.MagicManaCost.addType(MagicManaCost.java:103)
        at magic.model.MagicManaCost.<init>(MagicManaCost.java:69)
        at magic.model.MagicManaCost.create(MagicManaCost.java:339)
        at magic.ui.MagicImages.getIcon(MagicImages.java:198)
        at magic.ui.cardBuilder.renderers.OracleText.textIconReplace(OracleText.java:360)
        at magic.ui.cardBuilder.renderers.OracleText.drawOracleText(OracleText.java:45)
        at magic.ui.cardBuilder.renderers.CardBuilder.makeBasicCard(CardBuilder.java:91)
        at magic.ui.cardBuilder.renderers.CardBuilder.getCardBuilderImage(CardBuilder.java:32)
        at magic.ui.MagicImages.getMissingCardImage(MagicImages.java:253)
        at magic.test.TestCardBuilder.getGame(TestCardBuilder.java:18)
        at magic.test.TestGameBuilder.buildGame(TestGameBuilder.java:97)
        at magic.MagicMain.startUI(MagicMain.java:107)
        at magic.MagicMain.access$000(MagicMain.java:22)
        at magic.MagicMain$1.run(MagicMain.java:53)
        at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
        at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:756)
        at java.awt.EventQueue.access$500(EventQueue.java:97)
        at java.awt.EventQueue$3.run(EventQueue.java:709)
        at java.awt.EventQueue$3.run(EventQueue.java:703)
        at java.security.AccessController.doPrivileged(Native Method)
        at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:76)
        at java.awt.EventQueue.dispatchEvent(EventQueue.java:726)
        at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
        at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
        at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
        at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
        at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
        at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

**Comment by melvinzhang** [comment]
@ShawnieBoy need to add an icon for generic {P} without no circle behind it as shown on Rage Extractor, see http://magiccards.info/scans/en/nph/91.jpg

**Comment by ShawnieBoy** [comment]
Fixed after some confusion: 10374b5247f81713ed017ed1cb5ca5950f09bd4a 135c25ffaffd2e01c6f0905b83a4f90473982140 8f20288e3601e12c8505f64620f98f00fa43cc8b
